core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceRepository.scala

```diff
@@ -21,6 +32,26 @@ class WorkspaceRepository(dataSource: SlickDataSource) {
       access.workspaceQuery.findById(workspaceId.toString)
     }
 
+  def getWorkspaceContext(workspaceName: WorkspaceName,
```

Comment:
🤔 I know this was nomenclature from `WorkspaceService` but: I'm wondering if we should make this an overload of `getWorkspace()` -- they're both returning `Future[Option[Workspace]]`. There doesn't seem to be anything context-ful about this method.  Thoughts?

Reply:
I didn't do that initially, because Scala doesn't allow multiple overloads with optional parameters.

Then I realized I could just use plain pass-through overloads for all of the combinations.